extensions/1password/src/v8/utils.ts

```diff
@@ -21,8 +22,11 @@ export class ExtensionError extends Error {
 }
 
 export class NotFoundError extends ExtensionError {}
+
 export class CommandLineMissingError extends ExtensionError {}
+
 export class ZshMissingError extends ExtensionError {}
+
 export class ConnectionError extends ExtensionError {}
```

Comment:
Can you revert these unrelated changes? Let's keep the original author's code style and only modify the code relevant to the pull request.

If every contributor formats the code once according to their preferences, then git blame will become very bad.

Reply:
👌 Thanks review